app/splits/[id]/split-game-summary.tsx

```diff
@@ -0,0 +1,29 @@
+import ChampCard from "@/app/components/champ-card";
+
+import { Center, Flex, Spacer, Text } from "@chakra-ui/react";
+
+function getWinStatus(win: boolean | null) {
```

Comment:
Although I like this as a boolean, I wonder if being something that can have 3 states might be better handled as an enum of 3 values

Win
Loss
Remake?

Thoughts?